dev-libs/libbase58/libbase58-0.1.4.recipe

```diff
@@ -0,0 +1,61 @@
+SUMMARY="A library for encoding/decoding base58"
+DESCRIPTION="libbase58 provides encoding and decoding for the base58 format, \
+used in Bitcoin addresses."
+HOMEPAGE="https://github.com/bitcoin/libbase58"
+COPYRIGHT="Luke Dashjr"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/bitcoin/libbase58/archive/v${portVersion}.tar.gz"
+CHECKSUM_SHA256="97e8c3387fd4ed90208a8101e658743efbb6e21714e594f0828e6414f92c0b1d"
+
+ARCHITECTURES="x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libbase58$secondaryArchSuffix = $portVersion
+	lib:libbase58$secondaryArchSuffix = 0.0.2
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libbase58${secondaryArchSuffix}_devel = $portVersion
+	devel:libbase58$secondaryArchSuffix = 0.0.2
+	"
+REQUIRES_devel="
+	libbase58$secondaryArchSuffix == $portVersion
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoreconf
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	"
+
+
+BUILD()
+{
+	./autogen.sh
+	runConfigure ./configure
+	make $jobArgs
+}
+
+INSTALL()
+{
+	make install
+	prepareInstalledDevelLib libbase58
+	fixPkgconfig
+	packageEntries devel $developDir $documentationDir
```

Comment:
Since the only files in $documentationDir are AUTHORS, COPYING and README, I think we can put them in the main package. To do so, you just need to remove the `$documentationDir` (and the space before it, of course) from this line.
Anything that is not "moved" to a sub-package gets stored in the main package.
We also like to split the line after "devel". (It's a tradition ;-)
```
	packageEntries devel \
		$developDir
```
The reason for this is that the first argument passed to packageEntries is the suffix for the subpackage, while all the other arguments are files or directories that need to be placed in that sub-package.